hero-ups.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: page
+title: Unified Push Server (UPS) on OpenShift
+permalink: /hero-ups/
+---
+
+# Unified Push Server (UPS) on OpenShift
+
+## Prerequisites ## 
+
+You should have a locally installed version of OpenShift, or access to an OpenShift cluster. If you don't have either, you can set up your own OpenShift local cluster by reading this [document](<insert-openshift-hero-link>).
+
+## Bringing up your cluster ## 
+
+Bring up your cluster with:  
+`oc cluster up --public-hostname=127.0.0.1 --host-data-dir=$HOME/os/data-dir --host-config-dir=$HOME/os/config-dir`  
+
+![oc-cluster-up][oc-cluster-up]
+
+## Log into your cluster ##
+
+Log into your cluster via `oc login`
+
+![oc-login][oc-login]
+
+## Deploy MySQL containers ##
+
+We will be using the UPS docker image from Docker Hub. This image requires that we have 2 MySQL instances named 'unifiedpush'  and 'keycloak'. Create these by:  
+
+* `oc new-app mysql MYSQL_USER=unifiedpush MYSQL_PASSWORD=unifiedpush MYSQL_DATABASE=keycloak --name=keycloak`  
+
+![deploy-mysql-keycloak][deploy-mysql-keycloak]  
+
+* `oc new-app mysql MYSQL_USER=unifiedpush MYSQL_PASSWORD=unifiedpush MYSQL_DATABASE=unifiedpush --name=unifiedpush`  
+
+![deploy-mysql-unified-push][deploy-mysql-unified-push]  
+
+Confirm that both MySQL instances have come online by checking the OpenShift console, or via the terminal with `oc status`.  
+
+![pod-keycloak][pod-keycloak]  
+![pod-unifiedpush][pod-unifiedpush]  
+
+## Deploy AeroGear UPS ##
+
+With both required MySQL instances installed and running, UPS can now be deployed by running the following in your terminal:
+
+* `oc new-app aerogear/unifiedpush-wildfly UNIFIEDPUSH_PORT_3306_TCP_ADDR=unifiedpush UNIFIEDPUSH_PORT_3306_TCP_PORT=3306 UNIFIEDPUSH_ENV_MYSQL_DATABASE=unifiedpush  KEYCLOAK_PORT_3306_TCP_ADDR=keycloak KEYCLOAK_PORT_3306_TCP_PORT=3306 KEYCLOAK_ENV_MYSQL_DATABASE=keycloak UNIFIEDPUSH_ENV_MYSQL_USER=unifiedpush  UNIFIEDPUSH_ENV_MYSQL_PASSWORD=unifiedpush KEYCLOAK_ENV_MYSQL_USER=unifiedpush KEYCLOAK_ENV_MYSQL_PASSWORD=unifiedpush --name=unifiedpush-wildfly`   
```

Comment:
I think this will be better preresented as a code block with a multiline command (you can use \ to escape the newliane)